addon/services/flash-messages.js

```diff
@@ -68,6 +68,13 @@ export default Service.extend({
     return this;
   },
 
+  getFlashObject() {
```

Comment:
@poteto Thoughts on if this is acceptable based on the existing pattern? Addresses #154 but it isn't quite as fluent.  Also, thoughts on renaming to `getLastFlashObject` since it is a public method and could be called anywhere?